lambda/matching.ml

```diff
@@ -3875,9 +3905,15 @@ let toplevel_handler ~scopes loc ~failer partial args cases compile_fun =
                       failure_handler ~scopes loc ~failer ())
   end
 
+let root_arg arg binding_kind =
+  (* The mutability information denotes the mutability of a *position*
+     inside the value, whether looking inside the value of the
+     scrutinee is a pure operation. At the root we are immutable. *)
```

Comment:
```suggestion
  (* The mutability information denotes the mutability of a *position*
     inside the value, which indicates whether looking inside the
     value of the scrutinee is a pure operation. At the root we are
     immutable. *)
```
"whether" can mean "regardless whether"; this rewording makes it clear that it doesn't mean that here.